scripts/hooks/pre-push.js

```diff
@@ -59,17 +59,15 @@ function getDateFromFirstCommit(fromSHA, toSHA) {
 /// MAIN
 ////////////////////////////////////////////////////////////////
 
-const [remoteName, remoteUrl] = process.env.HUSKY_GIT_PARAMS.split(' ')
-
+// must trim otherwise the name will be 'origin\n'
+const remoteName = execSync('git remote').toString().trim()
```

Comment:
is this wanted? the remote name came from husky before

Reply:
yes.

Reply:
i guess that on my for not saying why i made this change. HUSKY_GIT_PARAMS no longer exists